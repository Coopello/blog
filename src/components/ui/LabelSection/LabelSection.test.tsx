import renderer from "react-test-renderer";
import { LabelSection } from "./";

describe("ui/LabelSection", () => {
  it("Snap Shot", () => {
    const component = renderer.create(<LabelSection />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
