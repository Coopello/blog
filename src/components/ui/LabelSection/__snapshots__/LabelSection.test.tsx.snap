// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/LabelSection Snap Shot 1`] = `
<section
  className="flex w-full flex-col"
  style={
    {
      "gap": "1rem",
    }
  }
>
  <h2
    className="border-custom-primary px-3 py-1 text-custom-black sm:px-4 sm:py-2"
    style={
      {
        "borderLeftWidth": 8,
        "fontSize": "1.5rem",
        "lineHeight": "2rem",
      }
    }
  >
    記事を書いた人
  </h2>
  <div />
</section>
`;
